LinkedIdeas/CanvasViewController.swift

```diff
@@ -160,6 +221,22 @@ class CanvasViewController: NSViewController {
     return results
   }
   
+  func clickedElements(atPoint clickedPoint: NSPoint) -> [Element]? {
+    var results = [Element]()
+    let clickedConcepts: [Element] = document.concepts.filter { (concept) -> Bool in
+      return concept.contains(point: clickedPoint)
+    }
+    let clickedLinks: [Element] = document.links.filter { (link) -> Bool in
+      return link.contains(point: clickedPoint)
+    }
+    
+    results.append(contentsOf: clickedConcepts)
+    results.append(contentsOf: clickedLinks)
+    
+    guard results.count > 0 else { return nil }
```

Comment:
`SwiftLint` would go nuts over this, it hates checking count. It prefers `isEmpty` 😎